scout/server/blueprints/variant/controllers.py

```diff
@@ -486,26 +486,19 @@ def observations(store: MongoAdapter, loqusdb: LoqusDB, variant_obj: dict) -> Di
     loqus_query = loqusdb.get_loqus_query(variant_obj, category)
 
     for loqus_id in inst_loqus_ids:  # Loop over all loqusdb instances of an institute
-        obs_data[loqus_id] = {}
-        loqus_settings = loqusdb.loqusdb_settings.get(loqus_id)
-
-        if loqus_settings is None:  # An instance might have been renamed or removed
+        # collect observation on that loqusdb instance
+        obs_data[loqus_id] = loqusdb.get_variant(loqus_query, loqusdb_id=loqus_id)
+        if obs_data[loqus_id] is None:
```

Comment:
Hm, no? `obs_data[loqus_id]` will also not be set when there is no frequency data for the variant!

Reply:
Compare:
```
[hiseq.clinical@cg-services-stage ~]$ curl "http://cg-services-prod.scilifelab.se:7102/random_stuff"
{"detail":"Not Found"}
[hiseq.clinical@cg-services-stage ~]$ curl "http://cg-services-prod.scilifelab.se:7102/variants/9_107584945_C_G"
{"detail":"Variant 9_107584945_C_G not found"}
[hiseq.clinical@cg-services-stage ~]$ curl "http://cg-services-prod.scilifelab.se:7102/variants/9_107584945_C_A"
{"chrom":"9","observations":1,"families":["hugeturtle"],"_id":"9_107584945_C_A","start":107584945,"end":107584946,"ref":"C","alt":"A","homozygote":0,"hemizygote":0,"total":459}
```

Reply:
Right! I even had to modify the test.. I also want to cover the case when the instance is not reachable at all. Perhaps I can move the alert inside the extension itself. I'll fix, thanks!